Tween/AnimationCollection.cs

```diff
@@ -37,7 +37,7 @@ public AnimationCollection Add(params AbstractAnimation[] animations)
 		/// </summary>
 		/// <param name="onComplete">An optional callback invoked when the AnimationCollection is complete</param>
 		/// <param name="onAbort">An optional callback invoked if the AnimationCollection is aborted</param>
-		public override void Play(Action onComplete = null, Action onAbort = null)
```

Comment:
Why the change to a non-optional callback on complete? If you want to keep that change, the documentation above the method should change as well.

Reply:
Because the base class has overload functions like `Play() { this.Play(null); }` It makes the default parameters become pointless.

And yes I'll update the doc.